.github/workflows/ci.yml

```diff
@@ -4,9 +4,18 @@ name: Go
 on:
   push:
     branches: main
-  pull_request: # don't include branches so that all PRs run.
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
```

Comment:
Is this an intentional change, if so, can you add some context for why we need to change this?

Reply:
sure.  the reason is that `labeled` and `unlabeled` are not included in the default list of types to run CI on and so if you add the `skip-changelog` or `ignore-release` label it won't re-run CI.  i'll add a comment in code